apps/dashboard/app/models/concerns/support_ticket_validator.rb

```diff
@@ -42,15 +30,15 @@ def validate_attachments(attribute)
       return
     end
 
-    if attribute.value.size > SupportTicketValidator.restrictions[:max_items]
-      errors.add(attribute.id, I18n.t('dashboard.support_ticket.validation.attachments_items', id: attribute.id, items: attribute.value.size, max: SupportTicketValidator.restrictions[:max_items]))
+    if attribute.value.size > @restrictions[:max_items]
```

Comment:
Similar thing about using `attr_reader`s. Looks like the other model is using them.

Reply:
fixed